test/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,32 @@
+using Test
+
+using MathOptInterface
+const MOI = MathOptInterface
+const MOIT = MathOptInterface.Test
+const MOIU = MathOptInterface.Utilities
+const MOIB = MathOptInterface.Bridges
+
+include("../utilities.jl")
+
+mock = MOIU.MockOptimizer(MOIU.Model{Float64}())
+config = MOIT.TestConfig(duals = false)
+
+@testset "NormInfinity" begin
+    mock.optimize! = (mock::MOIU.MockOptimizer) -> MOIU.mock_optimize!(mock, [1.0, 0.5, 1.0])
+    bridged_mock = MOIB.Constraint.NormInfinity{Float64}(mock)
+    MOIT.norminf1vtest(bridged_mock, config)
+    MOIT.norminf1ftest(bridged_mock, config)
+    # Dual is not yet implemented for NormInfinity bridge
+    ci = first(MOI.get(bridged_mock, MOI.ListOfConstraintIndices{MOI.VectorAffineFunction{Float64}, MOI.NormInfinityCone}()))
+    test_delete_bridge(bridged_mock, ci, 3, ((MOI.VectorAffineFunction{Float64}, MOI.Nonnegatives, 0),))
```

Comment:
Needs to test with `test_model_equals`, see https://github.com/JuliaOpt/MathOptInterface.jl/issues/820

Reply:
I'm going to need to wait to see more examples of how that is supposed to look

Reply:
See https://github.com/JuliaOpt/MathOptInterface.jl/pull/821/commits/ae71c8b6e3a5f189f63513dd11cbc145bc21b6c7

Reply:
I would like to see tests with `test_models_equal`. I find these really helpful to see concretely what the bridge is doing.

Reply:
OK. does that mean tests in the style of both https://github.com/JuliaOpt/MathOptInterface.jl/blob/d9673e8c04f23519436a747cbc8aeb881235ceae/test/Bridges/Variable/rsoc_to_psd.jl#L26 and https://github.com/JuliaOpt/MathOptInterface.jl/blob/d9673e8c04f23519436a747cbc8aeb881235ceae/test/Bridges/Variable/rsoc_to_psd.jl#L61? what are the high-level purposes of those two testsets?

@blegat those are for a variable bridge instead of a constraint bridge like in this PR. how different do they need to look for constraint bridge? do we have any closer examples I can use as a template here?

Reply:
done